### ISSUE TITLE
Participation curriculum goals for leadership are achieved and ready for testing in next heartbeat.

### ISSUE THREAD

**Issue by emmairwin** [mozilla/participation-org#351]
## Goal:

Build strong learning resources and opportunities to benefit participation events and projects.
## Success:
- A light-weight,  online "Train the Trainer" workshop is created to help prepare community leaders for the opportunity, and challenge of running events & workshops for their community.  (run next heartbeat)
-  2 workshops are identified that can bring value to the Connected Devices and Campus Campaign goals ( one each)
-  5 - 6 sessions/workshops from the Summit are prioritized, polished and packaged for community replication including an interaction of the Summit event arc(influenced by goals for Brazil meetup).
## Related Issues & Links:
#286
#371
#356
## Roles:

Owner: @emmairwin
Involved/consulted: @geroter @brianking @rosanardila @deimidis @williamq  @nukeador @flukeout @hannahkane 
## Volunteer Roles
- Help porting our chosen session workshops into polished format using our new template (available Feb 23rd)
## Required:
- [x] @emmairwin, identify 5-6 workshops to be developed and tested in Brazil ( [small], Due: Feb 24) 
- [x] @emmairwin, connect with Guillermo, Rosana and Ruben to ensure curriculum learning outcomes align with those needed in Brazil l ([small], Due: Feb 24) 
- [x] @emmairwin,Polish those 5-6 workshops using templates provided by curriculum team for Participation.l ( [large], Due: March 9th) 
- [x] @emmairwin, Identify and draft 2 workshops that can bring value to Campus Campaign & CD in consultation with leads in those areas. l ([medium], Due: March 10th) 
- [ ] @emmairwin, draft of a online 'train the trainer' workshop is created for testing in the next heartbeat - leveraging resources and input of curriculum team. l ([medium], Due: March 10th)

**Comment by emmairwin** [comment]
Identified Workshops
- Mentoring & Coaching  
- Unconscious Bias  cc/@larissashapiro
- Personal Leadership
- Ideas to Action
- Rapid Prototyping
- Running an Awesome Hackathon
- The Art of Community Communication

**Comment by flukeout** [comment]
Here is the **Rapid Prototyping** workshop content converted into the template...
http://flukeout.github.io/participation-org/rapid-prototyping/

Let me know your thoughts so far.

Right now everything in my own fork of this repo, but once we're closer to a final design and style I can submit a pull request here.

Original content is [here](https://wiki.mozilla.org/Participation/Global_Gatherings_2015/Singapore/Content/rapid_prototyping).

@emmairwin Do you have a copy of the Participation logo in vector format?

**Comment by flukeout** [comment]
PR is #379

**Comment by emmairwin** [comment]
Thanks @flukeout sorry I haven't managed a higher resolution logo!   So appreciate this !

**Comment by emmairwin** [comment]
Repo for Participation Curriculum:  https://github.com/mozilla/participation-curriculum

**Comment by emmairwin** [comment]
![mozilla_leadership_curriculum_personal_leadership_becoming_a_mozillian_-_2016-03-01_10 38 28](https://cloud.githubusercontent.com/assets/60618/13437584/e79cdedc-df99-11e5-9a0f-14810092c4bd.png)
![mozilla_leadership_curriculum_personal_leadership_becoming_a_mozillian_-_2016-03-01_10 37 53](https://cloud.githubusercontent.com/assets/60618/13437585/e7bd2714-df99-11e5-876f-e24d886ffce4.png)

@flukeout , I would like to have the same theme components for time, and tagging (web literacies)  as chad has in this template.

http://chadsansing.github.io/leadership-curriculum/1-1-becoming-a-mozillian.html#overview

are we going to standardize these types of page components as well?  ( I can go ahead and make these myself, but wondered).  Thanks  cc/  @chadsansing

**Comment by flukeout** [comment]
@emmairwin Because the content.md file is created with markdown, there's no easy way to indicate that a specific piece of content (like the time) should get a different styling, so this will be a bit of a challenge. 

Let me dig in and see what options we have, though.

**Comment by flukeout** [comment]
It looks like we can mix in HTML elements with the markdown content, so one idea would be to use a `<time />` tag for indicating times, like this...

![image](https://cloud.githubusercontent.com/assets/25212/13445263/08921532-dfc0-11e5-9cd8-997623888e9a.png)

Then we can apply styling when rendering the doc.

@emmairwin Let me know how that sounds. If you're into the idea, you can go ahead and start using that convention right now. It will just appear as a regular paragraph until I made the changes to the template CSS though, but you're not blocked on editing the content.

**Comment by emmairwin** [comment]
Hi @flukeout  - thanks so much, this works for me!

**Comment by emmairwin** [comment]
Personal Leadership: http://mozilla.github.io/participation-curriculum/personal-leadership/index.html needs QA review, but almost there.  

Also note I am using the http://mozilla/participation-curriculum/ repo

**Comment by emmairwin** [comment]
Theming needs to happen for 
- Time (length of workshop)
- Print (print page)
- Tag  (taxonomy terms)

and probably font awesome added so we can hopefully use icons in curriculum.

**Comment by jgmac1106** [comment]
Can we get the nav to be collapsed by default on mobile? Here is my view on FF for Android.
![screenshot_2016-03-02-18-46-09](https://cloud.githubusercontent.com/assets/10534566/13479840/c3d16e8c-e0a7-11e5-96cb-4064949bca7d.png)

**Comment by emmairwin** [comment]
@flukeout can say better. It might that I have more side nav than expected.  I wonder what a best practice would be for that.

**Comment by flukeout** [comment]
@jgmac1106 I'll file a separate issue for mobile viewing, where I agree that the navigation isn't all that useful.

**Comment by emmairwin** [comment]
One question I'll put out there for others reviewing this content is how can we 'teach facilitators' to deliver this content, as part of the content itself. Is that a Facilitators Intro video?  Or would each section warrant a short intro video to contextualize the skills, and opportunity of the group interactions, discussion.  any examples of this being done well?  Thanks

**Comment by emmairwin** [comment]
One more thing - this is our list of content we're polishing for a content 'bundle'.  many are still stubs, but working on them this week

http://mozilla.github.io/participation-curriculum/

**Comment by flukeout** [comment]
@emmairwin Is the new home for the template asset files the "participation curriculum" repo? I'd like to keep that stuff up-to-date in one place only if possible.

**Comment by flukeout** [comment]
I've filed the requests for new styling and mobile view as issues in the participation-curriculum repo.

**Comment by jgmac1106** [comment]
@emmairwin You can close this ticket back down and start one specifically for (or point me to) one that is specific to curriculum review.

I would keep the video to a minimum for data sensitive customers. If you are going to do an illustrative intro a gif or a a logical icon might work on the inner pages. 

Not a FOSS solution but I love videoscribe for narrated animations anyone can make. 

"In Plain English" genre (though with a more inclusive title) genre are easy and stick figures play well with l10n.

If you start with an intro one make it under three minutes 480p and use a predictable set of bumpers throughout (I go back and steal from Mayer's, dated but useful Multimedia Learning). Make the assets publicly available and hopefully the trained trainers start making stuff to train more trainers. 

Since we are  shooting for a "train the trainers" model  the logical thing would be to run the trainers through the the training.

Given that the goal is a self-directed leadership curriculum we can  look at the activities  carefully and provide feedback. Long term what a self-directed will have to have are artifacts that can serve as mentor texts. Trainers will need to see what other trainers have done.

**Comment by emmairwin** [comment]
@flukeout yes , the participation curriculum repo should be our single source for now.  Thanks for help with everything!

**Comment by emmairwin** [comment]
@jgmac1106 "Since we are shooting for a "train the trainers" model the logical thing would be to run the trainers through the the training."
I think this is the route.

**Comment by emmairwin** [comment]
redirected http://education.mozilla-community.org/  to this new repo.
personal leadership, rapid prototyping, design thinking for connected devices and interviewing users for Mozilla are ready for testing (although I will go over them again tomorrow for polishing).  How to run an  awesome hackathon is still in draft.  

I'm also happy to see Open Science   remix and polish this workshop based on our participation one.  http://flukeout.github.io/science-template/

**Comment by emmairwin** [comment]
I did not get to train the trainer this heartbeat - as want to talk more with Amira about leveraging the platform they're building.

Final note here - we need these resources to download as PDF, or we will have to move them.
Talking to Hannah and team about hopefully building this into what we have, vrs something separate.

**Comment by flukeout** [comment]
@emmairwin I've filed a follow-up issue in https://github.com/mozilla/participation-curriculum/issues/6 - Please check it out and leave your thoughts there.
